object_detection/README.md

```diff
@@ -35,7 +35,7 @@ Before You Start:
 * <a href='g3doc/installation.md'>Installation</a><br>
 
 Quick Start:
-* <a href='object_detection_tutorial.ipynb'>
```

Comment:
I still believe this change should be reverted, but your points are also valid. IMO, the correct thing to do is to revert this change, but update the notebook to be self-documenting (add a section about installation and running to the notebook). Changing the notebook should be done in another PR though.